controllers/apirule_controller.go

```diff
@@ -63,12 +95,9 @@ type APIRuleReconciler struct {
 //+kubebuilder:rbac:groups="apiextensions.k8s.io",resources=customresourcedefinitions,verbs=get;list;watch;create;update;patch;delete
 
 func (r *APIRuleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	//_ = r.Log.WithValues("Api", req.NamespacedName)
-
 	r.Log.Info("Starting reconcilation")
 
 	api := &gatewayv1beta1.APIRule{}
-
 	err := r.Client.Get(ctx, req.NamespacedName, api)
```

Comment:
This will probably produce an error on ConfigMap event

Reply:
yes, this will produce error. will fix it, need to check what resource/request is being reconciled.